src/Microsoft.Fast.Components.FluentUI/Components/Icons/CoreIcons.cs

```diff
@@ -0,0 +1,58 @@
+// <auto-generated>
+//     This code was generated by a tool.
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+
+#pragma warning disable 1591
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+internal static partial class CoreIcons
```

Comment:
Do these icons exist twice (here and in lib)?

Reply:
Yes: here to be independent of the optional Icon lib (but `internal`) and in the public Icon lib.